content/graphing/infrastructure/process.md

```diff
@@ -92,6 +92,39 @@ In the [dd-agent.yaml][12] manifest used to create the daemonset, add the follow
 
 Refer to the standard [daemonset installation][13] and the [Docker Agent][11] information pages for further documentation.
 
+### Process Arguments Scrubbing
+
+In order to avoid that sensitive data is sent to Datadog servers and displayed on the Live Processes page, a Data Scrubber tool is shipped with the Process Agent. This feature is enabled by default and any process argument that matches one of the following words has its value hidden.
```

Comment:
Change this to `In order to hide sensitive data on the Live Process page`

Reply:
I would also rephrase "Data Scrubber tool" as it's not actually a separate tool. Maybe just `In order to hide... the process-agent will scrub arguments from the command line`